integration/integration_test.go

```diff
@@ -75,10 +79,22 @@ func testWithQEMU(t *testing.T, uinitPkg string) (string, *qemu.QEMU) {
 		t.Fatal(err)
 	}
 
+	// Copy the kernel into the temp directory.
+	kernel := filepath.Join(tmpDir, "bzImage")
+	if err := exec.Command("cp", "testdata/bzImage_amd64", kernel).Run(); err != nil {
```

Comment:
I would sort of prefer if you can that you just do this with two opens and an io.Copy. It's just easier not to depend on things like cp existing, and you can far better error reporting anyway. Could you write a function named cp?

Reply:
Ack, I'll create a `pkg/cp` package.

Reply:
I created a `cp` package, but now I don't actually need it. Consider it a freebie :)

Reply:
Never mind, it's actually useful.